packages/manager/components/web-property/spaProperty.tsx

```diff
@@ -26,7 +26,7 @@ const SPAProperty: FunctionComponent<Properties> = ({ webprop }: Properties) =>
             >
               <StyledCard>
                 <CardTitle>{prop.spaName}</CardTitle>
-                <CardBody>{prop.propertyName}</CardBody>
+                <CardBody></CardBody>
```

Comment:
@soumyadip007 use height instead of empty cardBody

Reply:
Okay, will do that later.